generators/boilerplate/templates/index.js

```diff
@@ -1,7 +1,7 @@
-'use strict';
 <% if (babel) { -%>
 export default {};
 <% } else { -%>
+'use strict';
```

Comment:
ESLint error in ES2015 with `'use strict';`
```
lib\index.js
  1:0  error  Unnecessary "use strict" directive  strict
```
Refering to the this http://people.mozilla.org/~jorendorff/es6-draft.html#sec-strict-mode-code

>Module code is always strict mode code.